api/database/seeders/PoolCandidateFilterSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\PoolCandidateFilter;
+use Illuminate\Database\Seeder;
+
+class PoolCandidateFilterSeeder extends Seeder
```

Comment:
Right now, I don't see any reason to seed Filters that aren't connected to a Request